### PR TITLE
Task swe 1445 create alphabet secondary screen

### PR DESCRIPTION
**Pull request by whoiam460**

### CODE REVIEW

**Review by olenashevchuk — APPROVED** [review]

File: src/app/components/bundles/ContentScreen/components/ContentScreenContent/ContentScreenContent.js [hunk 1]
```diff
@@ -4,7 +4,11 @@ import PropTypes from 'prop-types'
 const ContentScreenContent = (props) => {
   const { content } = props
 
-  return <Col cw={8}>{content}</Col>
+  return (
+    <Col pl={136} mt={60} cw={8}>
```
Comment (Olexandr25) [inline src/app/components/bundles/ContentScreen/components/ContentScreenContent/ContentScreenContent.js:8]: don't move Col
move Box inside Col

File: src/app/components/bundles/ContentScreen/components/ContentScreenInfo/ContentScreenInfo.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ const ContentScreenInfo = (props) => {
   const { title, subtitle, description, chapter, domainName } = props
 
   return (
-    <Col cw={4}>
+    <Col mt={60} p={0} cw={4}>
```
Comment (Olexandr25) [inline src/app/components/bundles/ContentScreen/components/ContentScreenInfo/ContentScreenInfo.js:7]: the same

File: src/app/screens/Typography/TypographyContent/SecondaryScreen.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { Alphabet } from 'app/components'
+import { ContentScreen } from 'components'
+
+const SecondaryScreen = () => {
+  return (
+    <>
```
Comment (Olexandr25) [inline src/app/screens/Typography/TypographyContent/SecondaryScreen.js:6]: useless Fragment

File: src/app/components/bundles/ContentScreen/components/ContentScreenContainer/ContentScreenContainer.styled.js [hunk 4]
```diff
@@ -3,5 +3,6 @@ import styled from 'styled-components'
 
 const ContentScreenContainerStyled = styled(Container)`
   flex: 1 0 auto;
+  margin-left: 96px;
```
Comment (aleksandrgolyk) [inline src/app/components/bundles/ContentScreen/components/ContentScreenContainer/ContentScreenContainer.styled.js:6]: is it possible not to use this strict value ? what will be with yours template on another mediasizes ?

File: src/app/components/bundles/ContentScreen/components/ContentScreenContent/ContentScreenContent.js [hunk 5]
```diff
@@ -1,10 +1,17 @@
...
+
 import PropTypes from 'prop-types'
 
 const ContentScreenContent = (props) => {
   const { content } = props
 
-  return <Col cw={8}>{content}</Col>
+  return (
+    <Col v="center" cw={8}>
+      <Box mt={4} pl={136}>
```
Comment (aleksandrgolyk) [inline src/app/components/bundles/ContentScreen/components/ContentScreenContent/ContentScreenContent.js:10]: is it possible not to use this strict value? what will be with your template on other media sizes ?

File: src/styles/vars/grid.css [hunk 6]
```diff
@@ -9,12 +9,12 @@
...
 
-    --brandbook-module-grid-gutter-default: 112px;
+    --brandbook-module-grid-gutter-default: 16px;
     --brandbook-module-grid-gutter-sm: 8px;
     --brandbook-module-grid-gutter-md: 12px;
     --brandbook-module-grid-gutter-lg: 16px;
-    --brandbook-module-grid-gutter-xl: 112px;
-    --brandbook-module-grid-gutter-xxl: 112px;
+    --brandbook-module-grid-gutter-xl: 16px;
+    --brandbook-module-grid-gutter-xxl: 16px;
```
Comment (aleksandrgolyk) [inline src/styles/vars/grid.css:17]: xl and xxl are the same? looks like typing error ?

**Review by aleksandrgolyk — CHANGES_REQUESTED** [review]

File: src/app/components/bundles/ContentScreen/components/ContentScreenContent/ContentScreenContent.js (hunk 5 above)
Comment (Olexandr25) [reply src/app/components/bundles/ContentScreen/components/ContentScreenContent/ContentScreenContent.js:10]: At this moment, we have strict values of screen, and work with them

File: src/app/components/bundles/ContentScreen/components/ContentScreenContainer/ContentScreenContainer.styled.js (hunk 4 above)
Comment (whoiam460) [reply src/app/components/bundles/ContentScreen/components/ContentScreenContainer/ContentScreenContainer.styled.js]: now we dont have responsive and different media size just this strict value

**Review by aleksandrgolyk — APPROVED** [review]

**Pull request merged**